sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml

```diff
@@ -6,6 +6,7 @@ azure:
     user-group:
```

Comment:
I don't think you should push to ZhuXiaoBing-cn:master
![image](https://user-images.githubusercontent.com/4465723/116526559-4e1a9a00-a90c-11eb-8b6c-c3420fc5a690.png)

Reply:
get it